locales/en-US/server.ftl

```diff
@@ -185,6 +185,8 @@ annotationUndoButton =
     .title = Undo
 annotationRedoButton =
     .title = Redo
+annotationTextButton =
+    .title = Add Text
```

Comment:
Just noticed that other `title` attributes use sentence case, so this should probably be "Add text"? (likely on @johngruen to answer)

Reply:
yep

Reply:
Done